stories/ProductTour.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, number, text, withKnobs } from '@storybook/addon-knobs';
+import { ProductTour } from '@textkernel/oneui';
+import { slides } from '../src/components/ProductTour/__mocks__/slides';
+
+storiesOf('Organisms|ProductTour', module)
+    .addDecorator(withKnobs)
+    .add('ProductTour', () => {
+        const handleCancel = (isChecked = false) => {
+            console.log(`onCancel was called with ${isChecked.toString()}`);
+        };
+        const handleFinished = (isChecked = false) => {
+            console.log(`onFinished was called with ${isChecked.toString()}`);
+        };
+
+        const numberOfSlides = number('Number of slides', 4, {
+            range: true,
+            min: 1,
+            max: 4,
+            step: 1,
+        });
+        const isMultiSlide = numberOfSlides > 1;
+
+        return (
+            <ProductTour
```

Comment:
Maybe consider making it possible to dismiss the modal in the story, otherwise I can't read the documentation behind the backdrop ;)

Reply:
You can :-) just click the `show tour` checkbox in the knobs. I think adding an actual implementation story for it will be misleading a bit - it is clearer this way that you have to control it from the outside.